### PR TITLE
fix: add missing extension for PR template

### PR DESCRIPTION
**Pull request by nschonni**
## Description

Add missing extension and fix minor Markdownlint issues now that it is parsed as MD

## Motivation

I don't always use these, but it looks like it was added some time ago, but isn't currently being picked up with the missing extension

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+## Related issues and pull requests
 
 <!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
 <!-- 👉 Highlight related pull requests using "Relates to #123" -->
 <!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->
 
-
 <!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->
+
+<!-- markdownlint-disable-file MD041-->
```
Comment (Josh-Cena) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: Can we simply disable Markdownlint for this file? The comment will be displayed to all authors, and that's not very helpful.

Comment (teoli2003) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: Yes, let's disable Markdonwlint for this file. It isn't really part of MDN proper and doesn't abide to the MDN style guide.

Comment (nschonni) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: Feel free to do a new PR without the disable

Comment (teoli2003) [inline .github/PULL_REQUEST_TEMPLATE.md:23]:
```suggestion
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
We don't want to add a magic comment here; we want to disable Markdownlint for this file as it is not part of MDN proper and doesn't have to follow the same rules.

**Comment by nschonni** [comment]
As mentioned on other PRs, @teoli2003 @Josh-Cena you two don't seem to like disable comments, but I'm not going to spend effort to work around your dislike of them

**Comment by Josh-Cena** [comment]
Just add the `.github` folder to https://github.com/mdn/content/blob/4f6ba2f5a1615fe155292cac416c7ab6b9d711ec/.markdownlint-cli2.jsonc#L160?

**Review by caugner — APPROVED** [review]

**Comment by wbamberg** [comment]
Closing in favour of https://github.com/mdn/content/pull/20940, hopefully.

**Pull request closed without merging**